beacon-chain/blockchain/testing/mock.go

```diff
@@ -448,5 +448,5 @@ func (s *ChainService) IsOptimistic(_ context.Context) (bool, error) {
 
 // IsOptimisticForRoot mocks the same method in the chain service.
 func (s *ChainService) IsOptimisticForRoot(_ context.Context, _ [32]byte, _ types.Slot) (bool, error) {
-	return false, nil
+	return s.Optimistic, nil
```

Comment:
Let's return false with todo and target this for `develop`. See:
https://github.com/prysmaticlabs/prysm/pull/10276

Reply:
This is a mock

Reply:
Yeap.. making it return `s.Optimistic` makes it easy to test.